cwms-data-api/src/test/resources/cwms/cda/data/dto/basin/basin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "basin-id": {
+    "office-id": "MVR",
+    "name": "TEST_LOCATION2"
+  },
+  "sort-order": 1.0,
+  "total-drainage-area": 1005.0,
+  "contributing-drainage-area": 1050.0,
```

Comment:
Total will be larger than contributing
 * @param p_total_drainage_area        The total area of the basin, including non-contributing drainage area
 * @param p_contributing_drainage_area The area of the basin that contributes flow to the primary stream